src/test/checksum/file1

```diff
@@ -1,2 +1,3 @@
 Hello.  This is a simple text file with a known checksum.
 The checksum unit test is run on this file to check for regressions.
+
```

Comment:
Okay, but changing it silently doesn't explain the issue or even really fix it, forcing poor people like me to ask you for details.  Maybe describe the change in the commit message?  Comment the README in the test directory explaining the size requirement?  Or better yet, just fix the test to round the file size down with a comment explaining why.